src/mobs/dialog.js

```diff
@@ -0,0 +1,35 @@
+import { floor1 } from '../world.js'
+import { Position } from '../chat.js'
+
+import { mobs } from './mobs.js'
+
+export default function dialog({ client }) {
+  const right_click = 2
+  client.on('use_entity', ({ target, mouse, sneaking }) => {
+    if (mouse === right_click && sneaking === false) {
+      if (target in floor1.mobs) {
+        const mob = floor1.mobs[target]
```

Comment:
Can you add a FIXME ?
```suggestion
        const mob = floor1.mobs[target] // FIXME: check proper entity id here
```